### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by SpoilerRules** [CCBlueX/LiquidBounce1.8-Issues#3850]
**Stuff**
A AutoBlock Option on KillAura called "NoPacketBlock"

**Introduction**
Don't send the packets like NoSlow "Packet" Option; That will don't send useless packets, so we can bypass Hypixel & CubeCraft AutoBlock Check and this is only don't send NoSlow's "Packet" Option Packets.

**TODO**
Bypass Hypixel & CubeCraft AutoBlock Check.

**Addional Content**
Bypass to AntiCheat's.

**Comment by yorik100** [comment]
Smartest idea SpoilerRules has ever had

**Comment by lolmc1982** [comment]
Very cul idea :)

**Comment by Litely1222** [comment]
idk but cul

**Comment by CzechHek** [comment]
The explanation doesn't make much sense.
It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?

**Comment by yorik100** [comment]
> 
> 
> The explanation doesn't make much sense.
> It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?
> 
> EDIT: He wants to disable NoSlow for AutoBlocking, because it still works when eating or using bow.

It sounds like it doesn't want to unblock and block everytime you attack with KillAura autoblock which makes perfect sense

**Comment by SpoilerRules** [comment]
> The explanation doesn't make much sense.
> It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?
> 
> EDIT: He wants to disable NoSlow for AutoBlocking, because it still works when eating or using bow.

no, i want to don't send noslow packet option packets while autoblocking

**Comment by yorik100** [comment]
> 
> 
> > The explanation doesn't make much sense.
> > It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?
> > EDIT: He wants to disable NoSlow for AutoBlocking, because it still works when eating or using bow.
> 
> no, i want to don't send noslow packet option packets while autoblocking

You're the type of person to prefer water over H2O besides both being the same

**Comment by CzechHek** [comment]
> > The explanation doesn't make much sense.
> > It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?
> > EDIT: He wants to disable NoSlow for AutoBlocking, because it still works when eating or using bow.
> 
> It sounds like it doesn't want to unblock and block everytime you attack with KillAura autoblock which makes perfect sense

He said that he doesn't want noslow to work when blocking, because it only works when eating or using bow, it autobans with autoblock. He uses lower blockrate to not get slowed down with sword.

**Comment by SpoilerRules** [comment]
> > > The explanation doesn't make much sense.
> > > It sounds like if you want to turn off NoSlow Packet option just for AutoBlock?
> > > EDIT: He wants to disable NoSlow for AutoBlocking, because it still works when eating or using bow.
> > 
> > 
> > It sounds like it doesn't want to unblock and block everytime you attack with KillAura autoblock which makes perfect sense
> 
> He said that he doesn't want noslow to work when blocking with sword, because it only works when eating or using bow, it autobans when blocking. He uses lower blockrate to not get slowed down.

yes

**Comment by aftery142** [comment]
you make no sense

**Comment by yorik100** [comment]
> 
> 
> you make no sense

**Comment by Aleaf-Egemen** [comment]
sounds like fake autoblock xdd

**Comment by SpoilerRules** [comment]
cubecraft removed noslow check, but still can be useful for hypixel..
